### PR TITLE
Update types.md

### PR DESCRIPTION
**Pull request by xfq**
Update the URI of standard built-in objects in JavaScript.

### CODE REVIEW

**Comment by xfq** [comment]
Done, thanks.

File: page/javascript-101/types.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ var bar2;
 
 ## Objects
 
-Everything else in JavaScript is considered an object. While there are [numerous built-in objects](https://developer.mozilla.org/en/JavaScript/Reference#Global_Objects, "MDN - Global Object Reference"), this chapter will cover:
+Everything else in JavaScript is considered an object. While there are [numerous built-in objects](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects "MDN - Standard built-in objects"), this chapter will cover:
```
Comment (arthurvr) [inline page/javascript-101/types.md]: `"MDN - Global Object Reference"` seems better than `"MDN - Standard built-in objects"` to me. (that's what was in originally)

Comment (xfq) [inline page/javascript-101/types.md]: I'm aware of that, but the title of the new page is "Standard built-in objects". If you think the old link is better, I should also change the link to [the right version of the old one](https://developer.mozilla.org/docs/Web/JavaScript/Reference#Global_Objects) (the old link without the extra comma and with language redirection) for consistency. WDYT?

Comment (arthurvr) [inline page/javascript-101/types.md]: Agreed. It's okay as the PR is now. If no people disagree I'll merge it in this evening.

**Pull request closed without merging**